Extra/Shaders/NewVegasReloaded/Shaders/SKYTEX.pso.hlsl

```diff
@@ -156,12 +156,12 @@ VS_OUTPUT main(VS_INPUT IN) {
         float3 cloudTint = lerp(pows(TESR_SkyLowColor.rgb, 5.0), sunColor, saturate(sunInfluence * saturate(greyScale))).rgb;
         cloudTint = lerp(cloudTint, white.rgb, sunHeight * isDayTime); // tint the clouds less when the sun is high in the sky
 
-        finalColor.rgb *= lerp(1.0, cloudTint * TESR_CloudData.w * 1.5, isDayTime); // cancel tint at night
+        finalColor.rgb *= lerp(1.0, cloudTint * TESR_CloudData.w * 1.333 * TESR_DebugVar.z, isDayTime); // cancel tint at night
```

Comment:
it would be good to remove the debug vars from the shaders before release

Reply:
True, removed